packages/gatsby/src/redux/actions/internal.ts

```diff
@@ -350,3 +357,58 @@ export const deleteNodeManifests = (): IDeleteNodeManifests => {
     type: `DELETE_NODE_MANIFESTS`,
   }
 }
+
+export const createJobV2FromInternalJob = (
```

Comment:
Did anything actually change in this function with the original one except the plugin part?

Reply:
no, just action shape (`plugin` part) and that slight change to appease TS